### PR TITLE
[IMP] website: add new `s_striped_top` snippet

### PR DESCRIPTION
**Pull request by anso-odoo**
This commit introduces the new `s_striped_top` intro snippet.

Part of task-4077427
task-4077621

requires: https://github.com/odoo/design-themes/pull/850

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by stefanorigano — CHANGES_REQUESTED** [review]
Mostly ok @anso-odoo , just a little note in the code 😉
Thanks!

File: addons/website/views/snippets/s_striped_top.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_striped_top" name="Striped Top">
+    <section class="s_striped_top o_cc o_cc5 pt56 pb96" data-oe-shape-data="{'shape':'web_editor/Origins/04_001','colors':{'c3': '#FFFFFF'},'flip':['y'], 'showOnMobile':true}">
+        <div class="o_we_shape o_web_editor_Origins_04_001 o_shape_show_mobile" style="background-image: url(&quot;/web_editor/shape/web_editor%2FOrigins%2F04_001.svg?c3=%23FFFFFF&amp;flip=y&quot;); background-position: 50% 100%;"/>
+        <div class="container">
+            <div class="row flex-column-reverse flex-lg-row">
```
[the remark this answers is not in the document]
Comment (anso-odoo) [reply addons/website/views/snippets/s_striped_top.xml]: @stefanorigano indeed, fixed thanks. Does the rest look okay for you ? I changed the image as well

**Review by stefanorigano — APPROVED** [review]
Just perfect 👍 

Ready to move forward when design-themes is ready ;)

**Review by SergeBayet — APPROVED** [review]
@anso-odoo LGTM. Thanks you for explaining me the "order" stuff :)

**Comment by bso-odoo** [comment]
@robodoo delegate=SergeBayet

File: addons/website/views/snippets/s_striped_top.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        <div class="o_we_shape o_web_editor_Origins_04_001 o_shape_show_mobile" style="background-image: url(&quot;/web_editor/shape/web_editor%2FOrigins%2F04_001.svg?c3=%23FFFFFF&amp;flip=y&quot;); background-position: 50% 100%;"/>
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-6 pt24 pb24 order-lg-0" style="order: 2;">
+                    <img class="img img-fluid" src="/web/image/website.s_striped_top_default_image" alt=""/>
+                </div>
+                <div class="col-lg-6 pt24 pb24 order-lg-0" style="order: 1;">
+                    <h1>Experience the Future of Innovation in Every Interaction</h1>
+                    <p class="lead">Innovation transforms possibilities into reality.</p>
+                    <a t-att-href="cta_btn_href" class="btn btn-primary"><t t-esc="cta_btn_text">Read more</t></a>
```
Comment (SergeBayet) [inline addons/website/views/snippets/s_striped_top.xml:15]: @anso-odoo I missed the t-esc here. :(

Comment (anso-odoo) [reply addons/website/views/snippets/s_striped_top.xml]: Fixed ✅

**Comment by SergeBayet** [comment]
@robodoo r+

**Pull request closed without merging**